public/js/index.js

```diff
@@ -856,6 +858,8 @@ Index.resizableColumns = function () {
         document.removeEventListener('mousemove', resizeColumn);
         document.removeEventListener('mouseup', stopResize);
         document.body.style.cursor = 'default';
+
+        document.location.reload(true);
```

Comment:
This makes the behavior worse it seems, the columns seemed to resize and remember their size fine without needing a page reload here.  

In fact it's worse now, since there appears to be some kind of race condition that prevents the column size from being saved in localStorage before the page reloads.